Go/emitter/enhanced.go

```diff
@@ -0,0 +1,35 @@
+package emitter
+
+type AnyListener func(string, ...interface{})
+
+type Emitter interface {
+	On(name string, listener Listener)
+	Emit(name string, data ...interface{})
+}
+
+type EnhancedEmitter struct {
+	*EventEmitter
```

Comment:
Is there a reason to use a pointer here?

Reply:
No